packaging/windows/product.wxs.template

```diff
@@ -13,35 +13,35 @@
         <Property Id="MINIMUMBUILD" Value="1709" Secure="yes"></Property>
-        <Condition Message="CodeReady Containers requires the Windows 10 Fall Creators Update (version 1709) or newer.">
+        <Condition Message="Red Hat OpenShift Local requires the Windows 10 Fall Creators Update (version 1709) or newer.">
             <![CDATA[Installed OR (CURRENTBUILD > MINIMUMBUILD)]]>
         </Condition>
-        <Condition Message="CodeReady Containers requires at least 9GB of RAM to run. Aborting installation.">
+        <Condition Message="Red Hat OpenShift Local requires at least 9GB of RAM to run. Aborting installation.">
             <![CDATA[Installed OR (PhysicalMemory >= 9126)]]>
         </Condition>
         <Property Id="WINDOWSEDITION" Secure="yes">
             <RegistrySearch Id="WindowsEditionReg" Root="HKLM" Key="SOFTWARE\Microsoft\Windows NT\CurrentVersion" Name="EditionID" Type="raw" />
         </Property>
-        <Condition Message="CodeReady Containers cannot run on Windows Home edition">
+        <Condition Message="Red Hat OpenShift Local cannot run on Windows Home edition">
             <![CDATA[Installed OR (WINDOWSEDITION <> "Core")]]>
         </Condition>
 
         <Property Id="CRCINSTALLED" Secure="yes">
-            <RegistrySearch Id="CrcInstalledReg" Root="HKCU" Key="Software\Red Hat\CodeReady Containers" Name="installed" Type="raw" />
+            <RegistrySearch Id="CrcInstalledReg" Root="HKCU" Key="Software\Red Hat\Red Hat OpenShift Local" Name="installed" Type="raw" />
         </Property>
 
         <util:Group Id="CrcUsersGroup" Name="crc-users" />
         <Directory Id="TARGETDIR" Name="SourceDir">
             <Directory Id="ProgramFiles64Folder">
-                <Directory Id="INSTALLDIR" Name="CodeReady Containers">
+                <Directory Id="INSTALLDIR" Name="Red Hat OpenShift Local">
                     <Component Id="CrcExe" Guid="*">
                         <File Id="CrcExe" Source="SourceDir\crc.exe" KeyPath="yes" DiskId="1" />
                         <RemoveFile Id="RemoveInstallFiles" Name="*.*" On="uninstall" />
                     </Component>
                     <Component Id="AdminHelper" Guid="*">
                         <File Id="AdminHelper" Source="SourceDir\crc-admin-helper-windows.exe" KeyPath="yes" DiskId="1" />
-                        <ServiceInstall Name="CodeReadyContainersAdminHelper" Description="Perform administrative tasks for the user" Arguments="daemon" DisplayName="CodeReady Containers Admin Helper" ErrorControl="normal" Start="auto" Type="ownProcess" />
-                        <ServiceControl Id="StartAdminHelperService" Name="CodeReadyContainersAdminHelper" Start='install' Stop='both' Remove='uninstall' />
+                        <ServiceInstall Name="crcAdminHelper" Description="Perform administrative tasks for the user" Arguments="daemon" DisplayName="Red Hat OpenShift Local Admin Helper" ErrorControl="normal" Start="auto" Type="ownProcess" />

```

Comment:
https://github.com/code-ready/admin-helper/blob/5e6bd7a6fad7c738ec2d90a151b58d52c2c6e0c8/cmd/admin-helper/daemon.go#L59-L68 this is hard coded in the admin-helper repo but I think if we create the service ourself then we can name it as we want (do need testing on windows)

Also https://github.com/code-ready/crc/blob/2ca23cf0c1ad9be74a581b4184b443dd8a1f2b77/pkg/crc/constants/constants_windows.go#L9 needs to change it is used as part of preflight to make sure if service is running.

Reply:
> https://github.com/code-ready/admin-helper/blob/5e6bd7a6fad7c738ec2d90a151b58d52c2c6e0c8/cmd/admin-helper/daemon.go#L59-L68 this is hard coded in the admin-helper repo but I think if we create the service ourself then we can name it as we want (do need testing on windows)

yeah, this is fine, we install the service in the msi so it doesn't depend on the one defined in admin-helper repo

Reply:
will make this change.